l10n_ch_sepa_direct_debit/data/payment_type_sepa_sct.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data noupdate="0">
+
+
+<!-- TODO In the suitable_bank_types field, we should restrict to IBAN type and Rib+IBAN of l10n_fr_rib ? -->
```

Comment:
Same comment as in other module.
This is related to question about france localisation.